src/features/events/components/EventOverviewCard/EventOverviewCard.tsx

```diff
@@ -0,0 +1,186 @@
+import EditIcon from '@mui/icons-material/Edit';
+import LocationOnIcon from '@mui/icons-material/LocationOn';
+import OpenInNewIcon from '@mui/icons-material/OpenInNew';
+import {
+  Box,
+  Button,
+  Card,
+  ClickAwayListener,
+  Divider,
+  Link,
+  TextField,
+  Typography,
+} from '@mui/material';
+import { FC, useState } from 'react';
+
+import EventDataModel from 'features/events/models/EventDataModel';
+import { getWorkingUrl } from 'features/events/utils/getWorkingUrl';
+import messageIds from 'features/events/l10n/messageIds';
+import theme from 'theme';
+import UseEditEventOverviewCard from './useEditEventOverviewCard';
+import { useMessages } from 'core/i18n';
+import { ZetkinEvent } from 'utils/types/zetkin';
+import ZUIPreviewableInput from 'zui/ZUIPreviewableInput';
+
+type EventOverviewCardProps = {
+  editable: boolean;
+  element: ZetkinEvent;
+  model: EventDataModel;
+  onEditModeEnter: () => void;
+  onEditModeExit: () => void;
+};
+
+const EventOverviewCard: FC<EventOverviewCardProps> = ({
+  editable,
+  element,
+  model,
+  onEditModeEnter,
+  onEditModeExit,
+}) => {
+  const messages = useMessages(messageIds);
+  const [startDate, setStartDate] = useState(element.start_time);
+  const [endDate, setEndDate] = useState(element.end_time);
+  const [link, setLink] = useState(element.url);
+  const [infoText, setInfoText] = useState(element.info_text);
+
+  const { clickAwayProps, containerProps, previewableProps } =
+    UseEditEventOverviewCard({
+      editable,
+      onEditModeEnter,
+      onEditModeExit,
+      save: () => {
+        model.updateEventData({
+          end_time: endDate,
+          info_text: infoText,
+          start_time: startDate,
+          url: link,
+        });
+      },
+    });
+
+  return (
+    <ClickAwayListener {...clickAwayProps}>
+      <Box {...containerProps}>
+        <Card>
+          {!editable && (
+            <Box display="flex" justifyContent="flex-end" m={2}>
+              <Button startIcon={<EditIcon />} variant="outlined">
+                {messages.eventOverviewCard.editButton().toUpperCase()}
+              </Button>
+            </Box>
+          )}
+          <Box
```

Comment:
I think this entire box and it's children should be taken out of this PR, since they are just placeholders. I think it's better to have a PR that is fully functional (but missing some features) than to have dead fields in the PR, as that will confuse other developers.

Reply:
Have deleted it!